webmemo.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'csv'
+require 'cgi/util'
+
+class MemoFile
+  def self.read
+    memo_file = CSV.read('memofile.csv')
```

Comment:
2 単語なのであればファイル名も区切っておきましょう。

```suggestion
    memo_file = CSV.read('memo_file.csv')
```

Reply:
ファイル名直しました。